rpcs/src/methods/chain_head.rs

```diff
@@ -292,6 +292,128 @@ impl<T: RpcConfig> ChainHeadRpcMethods<T> {
             .await
     }
 
+    /// Fetch the block body (ie the extrinsics in the block) given its hash.
+    ///
+    /// Returns an array of the hexadecimal-encoded scale-encoded extrinsics found in the block,
+    /// or `None` if the block wasn't found.
+    pub async fn archive_v1_body(&self, block_hash: T::Hash) -> Result<Option<Vec<Bytes>>, Error> {
+        self.client
+            .request("archive_v1_body", rpc_params![block_hash])
+            .await
+    }
+
+    /// Call the `archive_v1_call` method and return the response.
+    pub async fn archive_v1_call(
```

Comment:
hmm, we could perhaps have a shared implementation for this for both `archive_v1_call` and `archive_unstable_call` but doesn't matter that much since we will remove the unstable stuff eventually :)

Reply:
Yeah, I contemplated that, but then it'd make it messier to remove them, whereas at the mo we can just delete the unstable ones nice and cleanly :)